OpenRA.Game/Actor.cs

```diff
@@ -110,6 +112,19 @@ internal Actor(World world, string name, TypeDictionary initDict)
 				return new Rectangle(offset.X, offset.Y, size.X, size.Y);
 			});
 
+			visualBounds = Exts.Lazy(() =>
+			{
+				var sd = Info.Traits.GetOrDefault<ISelectionDecorationsInfo>();
```

Comment:
I think it should default to `Selectable.Bounds` if there isn't a trait that implements `ISelectionDecorationsInfo`, or it doesn't specify visual bounds.